### PR TITLE
feat(prl:3693 : Done inital setup for google analytics)

### PR DESCRIPTION
**Pull request by SasankAchyuta9**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the keep-helm label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/cookie-banner/template.njk [hunk 1]
```diff
@@ -1,51 +1,67 @@
...
+  <p class="govuk-body">{{ cookiesLine1 }}</p>
+  <p class="govuk-body">{{ cookiesLine2 }}</p>
 {% endset %}
 
-<div class="global-cookie-message" id="cm-cookie-banner" >
-  {{ govukCookieBanner({
-    ariaLabel: cookiesHeading + " " + serviceName,
-    messages: [
+    {% set acceptHtml %}
+  <p class="govuk-body">You’ve accepted additional cookies. You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>
```
Comment (AlokDatta) [inline src/main/steps/cookie-banner/template.njk:9]: what about welsh for this?

File: src/main/steps/common/common.content.ts [hunk 2]
```diff
@@ -97,13 +97,13 @@ export const en = {
   cookiesHeading: 'Cookies on',
   cookiesLine1: 'We use some essential cookies to make this service work.',
   cookiesLine2:
-    'We’d also like to use analytics cookies so we can understand how you use the service and make improvements.',
+    'We’d like to set additional cookies so we can remember your settings, understand how people use the service and to improve government services.',
```
Comment (AlokDatta) [inline src/main/steps/common/common.content.ts:100]: do we have welsh for this?

File: src/main/assets/js/cookie.ts [hunk 3]
```diff
@@ -1,65 +1,42 @@
...
-}
+cookieManager.on('UserPreferencesLoaded', preferences => {
+  const dataLayer = window.dataLayer || [];
+  dataLayer.push({ event: 'Cookie Preferences', cookiePreferences: preferences });
+});
 
-function cookiePreferencesUpdated(cookieStatus) {
+cookieManager.on('UserPreferencesSaved', preferences => {
   const dataLayer = window.dataLayer || [];
+  console.log('I am saved', dataLayer);
```
Comment (vivek-sekhar) [inline src/main/assets/js/cookie.ts:10]: Remove console logs.

File: src/main/assets/js/data-layer.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const dataLayer = window.dataLayer || [];
+dataLayer.push({ event: 'Site language', language: document.documentElement.lang });
+console.log(dataLayer);
```
Comment (vivek-sekhar) [inline src/main/assets/js/data-layer.ts:3]: Remove console logs.

File: src/main/steps/common/common.content.ts [hunk 5]
```diff
@@ -97,13 +97,13 @@ export const en = {
...
   acceptAnalyticsCookies: 'Accept analytics cookies',
   rejectAnalyticsCookies: 'Reject analytics cookies',
   viewCookies: 'View cookies',
   hideMessage: 'Hide this message',
   cookiesConfirmationMessage:
-    '<p>You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>',
+    '<p class="govuk-body">You can <a class="govuk-link" href="/cookies">change your cookie settings</a> at any time.</p>',
   changeCookiesHeading: 'Change your cookie settings',
   allowAnalyticsCookies: 'Allow cookies that measure website use?',
   useAnalyticsCookies: 'Use cookies that measure my website use',
```
Comment (vivek-sekhar) [inline src/main/steps/common/common.content.ts:109]: Line 126 for Welsh has a TODO. why is the infrastructure missing?

File: src/main/steps/cookie-banner/template.njk (hunk 1 above)
Comment (SasankAchyuta9) [reply src/main/steps/cookie-banner/template.njk:9]: No Alok we dont have welsh for this. But I have given welsh infrastructure for this as of now.

File: src/main/assets/js/cookie.ts (hunk 3 above)
Comment (SasankAchyuta9) [reply src/main/assets/js/cookie.ts:10]: Removed Vivek

File: src/main/assets/js/data-layer.ts (hunk 4 above)
Comment (SasankAchyuta9) [reply src/main/assets/js/data-layer.ts:3]: Removed.

File: src/main/steps/common/common.content.ts (hunk 5 above)
Comment (SasankAchyuta9) [reply src/main/steps/common/common.content.ts:109]: Yes Vivek .. Right now added welsh infra for this .

**Pull request merged**